src/main/java/core/basesyntax/Ball.java

```diff
@@ -0,0 +1,11 @@
+package core.basesyntax;
+
+public class Ball {
+    private String color;
+    private int number;
+
+    public Ball(String color, int number) {
+        this.color = color;
+        this.number = number;
+    }
```

Comment:
Good! But let's also add `getters`, `setters` and `toString()` method.

See [here](https://www.jetbrains.com/help/idea/generating-code.html#generate-equals-hashcode) on how to generate a `toString` fast.